tangelo/linq/tests/test_translator_perf.py

```diff
@@ -47,20 +47,22 @@
 class PerfTranslatorTest(unittest.TestCase):
 
     def test_perf_operator(self):
-        """ Performance test with a reasonable large input for operator """
+        """ Performance test with a reasonable large input for operator.
+        Symbolic backends are not included in this test.
+        """
 
         print(f'\n[Performance Test :: linq operator format conversion]')
         print(f'\tInput size: n_qubits={n_qubits_op}, n_terms={n_terms}\n')
 
-        for f in FROM_TANGELO_OP:
+        for f in FROM_TANGELO_OP.keys() - symbolic_backends:
```

Comment:
Define a variable with this expression, reuse it all the time. Call it `perf_backends` or something.